### PR TITLE
Adopting V3 API in routes

### PR DESCRIPTION
**Pull request by radoslav-tomov**
Change adopts  V3 API in the following operations: 

- check 
- deleteOrphanedRoutes
- delete

### CODE REVIEW

File: cloudfoundry-operations/src/main/java/org/cloudfoundry/operations/routes/DefaultRoutes.java [hunk 1]
```diff
@@ -30,7 +30,7 @@
 import org.cloudfoundry.client.v2.routes.DeleteRouteResponse;
```
Comment (anthonydahanne) [inline cloudfoundry-operations/src/main/java/org/cloudfoundry/operations/routes/DefaultRoutes.java:30]: unused import

Comment (anthonydahanne) [reply cloudfoundry-operations/src/main/java/org/cloudfoundry/operations/routes/DefaultRoutes.java:30]: also line 61: `import static org.cloudfoundry.util.tuple.TupleUtils.predicate;`

File: cloudfoundry-operations/src/test/java/org/cloudfoundry/operations/routes/DefaultRoutesTest.java [hunk 2]
```diff
@@ -23,10 +23,10 @@
 import org.cloudfoundry.client.v2.applications.AssociateApplicationRouteRequest;
 import org.cloudfoundry.client.v2.applications.AssociateApplicationRouteResponse;
 import org.cloudfoundry.client.v2.applications.RemoveApplicationRouteRequest;
-import org.cloudfoundry.client.v2.jobs.ErrorDetails;
-import org.cloudfoundry.client.v2.jobs.GetJobRequest;
-import org.cloudfoundry.client.v2.jobs.GetJobResponse;
-import org.cloudfoundry.client.v2.jobs.JobEntity;
+
+import org.cloudfoundry.client.v3.jobs.GetJobRequest;
```
Comment (anthonydahanne) [inline cloudfoundry-operations/src/test/java/org/cloudfoundry/operations/routes/DefaultRoutesTest.java:27]: lots of unused imports here

**Review by anthonydahanne — CHANGES_REQUESTED** [review]
please keep reformatting in a standalone commit; it makes it really hard to review if changes and reformatting (please configure your IDE not to reformat every single class you updated) are bundled together; thank you!

File: cloudfoundry-operations/src/test/java/org/cloudfoundry/operations/routes/DefaultRoutesTest.java (hunk 2 above)
Comment (radoslav-tomov) [reply cloudfoundry-operations/src/test/java/org/cloudfoundry/operations/routes/DefaultRoutesTest.java:27]: @anthonydahanne  Thanks for checking it out. Sure, will do.

**Comment by radoslav-tomov** [comment]
@anthonydahanne  Apologies for the formatting. I've been using the VS Code and it's default styling is a bit off.   I've removed the unused imports and reformatted via the standard  Eclipse google java style.

**Comment by anthonydahanne** [comment]
> @anthonydahanne Apologies for the formatting. I've been using the VS Code and it's default styling is a bit off. I've removed the unused imports and reformatted via the standard Eclipse google java style.

Thanks for the effort, but unfortunately, a lot of unchanged code is still reformatted.

We need to enforce formatting using CI to make sure we won't be having this discussion anymore.

I'll let you know when we have it.

Thanks for your patience

**Comment by radoslav-tomov** [comment]
@anthonydahanne Ok. Thanks.

**Comment by beyhan** [comment]
This could be closed now because https://github.com/cloudfoundry/cf-java-client/pull/1217 introduced this change.

**Comment by pivotal-david-osullivan** [comment]
Closed via #1217

**Pull request closed without merging**